### PR TITLE
Limit page width to improve readability.

### PR DESCRIPTION
**Pull request by okkays**
- 🌎 As pointed out by Tina, some pages on ada.gov feel difficult to read.
- ⛔ Her theory is that this is because the text width is too large
- ✅ This commit explores shrinking that width to 64rem (the "small screen" max width used throughout the site)
- 🔮 This is a draft to evaluate whether this would make pages more readable - feedback is encouraged!

Before: scanning the line:

> Knowing when the 1991 or the 2010 ADA Standards apply to buildings and facilities is important in determining if your building or facility complies with the ADA. The [ADA Requirements: Effective Date and Compliance Date guide](https://archive.ada.gov/revised_effective_dates-2010.htm) helps to explain which version of the ADA Standards to use and when.

is difficult because the eye has to travel far on large screens.

image

After feels easier on the eyes:

image

### CODE REVIEW

**Comment by JackRyan1989** [comment]
@okkays Initial run through with screen reader seems good to me. No discernible difference between this change and live site.

File: _pages/cases.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ redirect_from:
   - /enforce.htm
   - /enforce_activities/
 sidenav: false
+compact: true
```
Comment (JackRyan1989) [inline _pages/cases.md:17]: Let's make sure this is documented in our wiki: https://github.com/usdoj-crt/beta-ada/wiki/Front-matter

**Review by JackRyan1989 — APPROVED** [review]
Nice. This looks great. No concerns!

File: _pages/cases.md (hunk 1 above)
Comment (okkays) [reply _pages/cases.md:17]: nice, thanks!

**Pull request merged**